### DISCUSSION TITLE
The replace functionality within the `-field` flag is not working correctly

### DISCUSSION THREAD

**Discussion by forgedhallpass** [projectdiscovery/uncover#116] [Ideas]
Taking the example from https://github.com/projectdiscovery/uncover#field-formatting

```sh
uncover -f https://ip:port/version
```

TODO:
* [ ] Some escaping option needed: It is not possible to create a URL with a static path pointing to any of the reserved words (`ip`, `host`, `port`) e.g. `http://10.10.10.10:1234/host`
* [ ] The replace logic also replaces substrings within different words: e.g. `-f https://ip:port/hipster` would also translate to something like `https://10.10.10.10:1234/h10.10.10.10ster` 
* [ ] Wrap placeholders with something like: `{{ip}}`

**Comment by ehsandeep** [comment]
Reverted the changes (https://github.com/projectdiscovery/uncover/pull/57) to fix this issue at https://github.com/projectdiscovery/uncover/commit/f5832ad0633f42b9fc7368e92d70aad8055baafd for UX reasons. This requires further discussion to develop a solution that doesn't change the current seamless behavior of using predefined fields.
